askbot/jinja2/macros.html

```diff
@@ -466,16 +467,19 @@ <h2 class="comment-title">{% trans %}Comments{% endtrans %}</h2>
     {% if current_sort_method == key_name + "-asc" %}{# "worst" first #}
         <a id="by_{{key_name}}"
            href="{{ search_state.change_sort(key_name+"-desc").full_url() }}"
+           rel="nofollow"
```

Comment:
Wow, something crazy while testing this - in the `tab_bar.html` file, I wasn't seeing the nofollows on the sorting links.

It turned out to be this line in `tab_bar.html`:

```
{% cache 0 "scope_sort_tabs" search_tags request.user author_name scope sort query context.page language_code %}
```

... wow, caching inside of template spaghetti is not helpful or necessary :<

I'll look for a way to disable template caching on a whole-app basis, which will make testing easier...

Reply:
https://stackoverflow.com/questions/11297873/how-to-disable-cache-for-development-in-django

Well, the top suggestion here is the 'standard way', and it crashed askrover :|

There are only 3 instances of `{% cache %}` in askrover:
```
src/askbot-devel/askbot/jinja2/widgets/related_tags.html
2:{% cache 0 "tags" tags tag_list_type query_string language_code %}

src/askbot-devel/askbot/jinja2/question.html
379:        {% cache long_time "thread-content-html" thread.id %}

src/askbot-devel/askbot/jinja2/main_page/tab_bar.html
2:{% cache 0 "scope_sort_tabs" search_tags request.user author_name scope sort query context.page language_code %}
```

I'll be on the lookout for those throughout the rest of this review...